sdk/include/reactos/mc/pciclass.mc

```diff
@@ -27,6 +34,9 @@ SCSI Controller
 Language=Polish
 Kontroler SCSI
 .
+Language=Romanian
+Dispozitiv de control al domeniilor SCSI
```

Comment:
„Dispozitiv de control SCSI” (i.e. drop „al domeniilor” when it's about a low level computing part or protocol; and given the fact that this entire file „pci class” refers to the [PCI (hardware) data bus](https://en.wikipedia.org/wiki/Peripheral_Component_Interconnect) class of pluggable hardware boards, it means dropping it for everything in here).